### PR TITLE
Issue 161: Note about stochastic test failure

### LINKED ISSUE #161: Possibility of reproducible stochastic test fails
In PR #160 we have included setting the seed in R and the seed passed to `brms` such that repeat runs of the integration tests produce the same answer.

It is possible that the code for the model changes, and repeat runs of the test will always fail (for example if it's a p-value of 0.01 then this will happen in 1% of cases). 

I don't know if there is anything that can be done about this. I'd say that one of these integration tests failing is still a good signal something is up, but if you have really checked then it could just be a random fail.

Could test this with changing the seed and record distribution of test fails.

I've labelled this as "question" but really it's more like "warning" or "something to bear in mind".

### PR DESCRIPTION
**Pull request by athowes**
## Description

This PR closes #161.

I've added a note about stochastic test fails to every name of a stochastic test. I'm open to better solutions than this.

## Checklist

- [ ] My PR is based on a package issue and I have explicitly linked it.
- [ ] I have included the target issue or issues in the PR title in the for Issue(s) *issue-numbers*: PR title
- [ ] I have read the [contribution guidelines](https://github.com/epinowcast/.github/blob/main/CONTRIBUTING.md).
- [ ] I have tested my changes locally.
- [ ] I have added or updated unit tests where necessary.
- [ ] I have updated the documentation if required.
- [ ] My code follows the established coding standards.
- [ ] I have added a news item linked to this PR.
- [ ] I have reviewed CI checks for this PR and addressed them as far as I am able.

### CODE REVIEW

File: tests/testthat/test-int-latent_individual.R [hunk 1]
```diff
@@ -1,3 +1,9 @@
+stochastic_test_message <- paste0(
```
Comment (seabbs) [inline tests/testthat/test-int-latent_individual.R:1]: I think as this is developer side I would be tempted by just adding a comment to this effect vs printing it into the test message?

Comment (athowes) [reply tests/testthat/test-int-latent_individual.R:1]: Yeah fair, just a comment at the top of the script?

Comment (athowes) [reply tests/testthat/test-int-latent_individual.R:1]: Printing it to the test message makes it clear which tests this applies to

Comment (athowes) [reply tests/testthat/test-int-latent_individual.R:1]: I could also put the comment inside each of the relevant tests

Comment (seabbs) [reply tests/testthat/test-int-latent_individual.R:1]: > I could also put the comment inside each of the relevant tests

So I think this is what I would do but I don't feel that strongly so your call on what you think is clearer/most useful

Comment (athowes) [reply tests/testthat/test-int-latent_individual.R:1]: I don't have strong preferences either. I slightly dislike copy pasting the same message, so perhaps put it at the top and then a note to refer to the comment in each function

Comment (athowes) [reply tests/testthat/test-int-latent_individual.R]: I've implemented that approach in c7638e5

**Review by seabbs — APPROVED** [review]

**Pull request merged**